src/combobox/src/Combobox.js

```diff
@@ -28,12 +28,19 @@ const Combobox = memo(function Combobox(props) {
 
   const [isOpenedByButton, setIsOpenedByButton] = useState(false)
 
-  const handleStateChange = changes => {
+  const handleStateChange = (changes, stateAndHelpers) => {
     if (Object.prototype.hasOwnProperty.call(changes, 'isOpen')) {
       if (!changes.isOpen) {
         setIsOpenedByButton(false)
       }
     }
+
+    if (
+      autocompleteProps &&
+      typeof autocompleteProps.onStateChange === 'function'
+    ) {
+      autocompleteProps.onStateChange(changes, stateAndHelpers)
```

Comment:
Just out of curiosity, where is this second argument being consumed from? Is this from Downshift itself?

Reply:
Yep.